unblob/logging.py

```diff
@@ -2,22 +2,45 @@
 from pathlib import Path
 
 import structlog
+from dissect.cstruct import cstruct, dumpstruct
 
 
 def format_hex(value: int):
     return f"0x{value:x}"
 
 
-def pretty_print_paths(extract_root: Path):
-    def add_path(logger, method_name: str, event_dict: structlog.types.EventDict):
+def pretty_print_types(extract_root: Path):
+    def convert_type(logger, method_name: str, event_dict: structlog.types.EventDict):
         use_absolute_path = event_dict.pop("_absolute_path", False)
         for key, value in event_dict.items():
             if isinstance(value, Path):
                 path = value if use_absolute_path else value.relative_to(extract_root)
                 event_dict[key] = str(path)
+
+            elif isinstance(value, cstruct):
```

Comment:
I really, really love this idea!

However, I wanted to see for myself, and it was not working. I check the `zip` handler's debug logs, they were in a "normal" format, so the `dumpstruct` was not running for them.

With this modification I could make it work:
```suggestion
            elif isinstance(value, cstruct) or isinstance(value, Instance):
```